### PR TITLE
Implementation of local logger in request and alerts, left to be adap…

### PR DESCRIPTION
**Pull request by everamal**
…ted to production

### CODE REVIEW

File: src/app/Services/alerts.service.ts [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
-      dismiss = "";
+      dismiss = "Close";
       time = 3000;
     } else if (alertType == "info") {
       dismiss = "Got it!";
       time = 10000;
     }
-    this._snackBar.open(alertMessage, dismiss, {
+    
+    this.logger.info("Response alert with Info: '" + alertMessage + "', of type '" + alertType + ", with a duration of " + time + " milliseconds");
```
Comment (Miguelsiqueiros) [inline src/app/Services/alerts.service.ts]: We should use string interpolation when we concat strings, example:
 this.logger.info(`Response alert with Info: ${alertMessage}, of type${alertType}, with a duration of${time}milliseconds`);

File: src/app/Services/checkIn.service.ts [hunk 2]
```diff
@@ -3,47 +3,56 @@ import { HttpClient, HttpParams } from '@angular/common/http';
...
   dailyDatasource() {
     if (environment.production == true) {
+      this.logger.info("Attempt to get daily Ranking" + "(GET in Production)");
       return this.httpClient.get(`${environment.apiUrl}users/daily-ladder`);
     }
     else {
+      this.logger.info("Attempt to get daily Ranking" + "(GET in Development)");
       return this.httpClient.get(`${environment.apiUrl}users/daily-ladder`);
     }
   }
```
Comment (Miguelsiqueiros) [inline src/app/Services/checkIn.service.ts]: Same as the comment above.

File: src/app/Services/checkIn.service.ts [hunk 3]
```diff
@@ -3,47 +3,56 @@ import { HttpClient, HttpParams } from '@angular/common/http';
...
   providedIn: 'root'
 })
 export class CheckInService {
 
-  constructor(private httpClient: HttpClient) { }
+  constructor(private httpClient: HttpClient, private logger: NGXLogger) { }
 
   checkIn(pin: number) {
     if (environment.production == true) {
+      this.logger.info("Attempt to Check In, with pin " + pin.toString() + " (POST in Production)");
```
Comment (Maldox) [inline src/app/Services/checkIn.service.ts]: The message displayed, in my opinion is not appropriate. In no matter you should show the pin that has been typed. Even if it's a mistake

Comment (Maldox) [reply src/app/Services/checkIn.service.ts]: We need to discuss it

File: src/app/components/register/register.component.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<h1 class="toCenter">Register</h1>
+<h1 class="toCenter" (click)="TestAlert()">Register</h1>
```
Comment (Maldox) [inline src/app/components/register/register.component.html]: cammelCase please

File: src/app/Services/checkIn.service.ts (hunk 2 above)
Comment (everamal) [reply src/app/Services/checkIn.service.ts]: Got it

File: src/app/Services/alerts.service.ts (hunk 1 above)
Comment (everamal) [reply src/app/Services/alerts.service.ts]: ok!

Comment (everamal) [reply src/app/Services/alerts.service.ts]: Done

File: src/app/Services/checkIn.service.ts (hunk 2 above)
Comment (everamal) [reply src/app/Services/checkIn.service.ts]: Done

File: src/app/components/register/register.component.html (hunk 4 above)
Comment (everamal) [reply src/app/components/register/register.component.html]: Done

File: src/app/Services/checkIn.service.ts (hunk 3 above)
Comment (everamal) [reply src/app/Services/checkIn.service.ts]: yes

Comment (everamal) [reply src/app/Services/checkIn.service.ts]: changed logger implementation

**Pull request merged**